sycl/test/basic_tests/image/image_write.cpp

```diff
@@ -0,0 +1,276 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env SYCL_BE=%sycl_be %t.out
```

Comment:
Shouldnt we have CPU/GPU/ACC_PLACEHOLDERS here to check on every available device?

Reply:
Good eye, I have updated this.